### PR TITLE
Fix function call replacement

### PR DESCRIPTION
**Pull request by jmythms**
Pull request overview
---------------------
 - Fixes a function call change that accidentally crept in: commit [3fc6b96](https://github.com/NREL/EnergyPlus/commit/3fc6b96bbdc465b82edc93b3f58ae2bf9607232b).

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

File: src/EnergyPlus/ScheduleManager.cc [hunk 1]
```diff
@@ -562,7 +562,7 @@ namespace ScheduleManager {
                                 CSVAllColumnNameAndValues[colCnt - 1] = timestepColumnValues;
                             }
                         } else {
-                            columnValue = std::stod(subString, nullptr);
+                            columnValue = UtilityRoutines::ProcessNumber(subString, errFlag);
```
Comment (jmythms) [inline src/EnergyPlus/ScheduleManager.cc:565]: It is worth noting that this version is faster on my local machine while having no diffs. 
Ran in 5.74 seconds. 
image

File: src/EnergyPlus/ScheduleManager.cc [hunk 2]
```diff
@@ -562,7 +562,7 @@ namespace ScheduleManager {
                                 CSVAllColumnNameAndValues[colCnt - 1] = timestepColumnValues;
                             }
                         } else {
-                            columnValue = std::stod(subString, nullptr);
```
Comment (jmythms) [inline src/EnergyPlus/ScheduleManager.cc:565]: It is worth noting that this function is faster on my local machine while having no diffs. 
Ran in 5.74 seconds. 
image

File: src/EnergyPlus/ScheduleManager.cc (hunk 1 above)
Comment (jmythms) [inline src/EnergyPlus/ScheduleManager.cc:565]: The old functionis slower and takes 8.68 seconds. 
image

File: src/EnergyPlus/ScheduleManager.cc (hunk 2 above)
Comment (jmythms) [reply src/EnergyPlus/ScheduleManager.cc:565]: But, we would need to handle errors here, and I can explore that if this is something that interests us.

**Comment by jmythms** [comment]
The failed unit test seems to be passing locally. I think that might be a CI issue.

File: src/EnergyPlus/ScheduleManager.cc (hunk 2 above)
Comment (mitchute) [reply src/EnergyPlus/ScheduleManager.cc:565]: Which file were you using here?

Comment (jmythms) [reply src/EnergyPlus/ScheduleManager.cc:565]: Oops, forgot to mention that. The `SolarShadingTest_ImportedShading.idf` file.

Comment (mitchute) [reply src/EnergyPlus/ScheduleManager.cc:565]: Umm, what is up with my computer??? 2 min 28 sec...

Comment (mitchute) [reply src/EnergyPlus/ScheduleManager.cc:565]: And with using with the ```std::stod``` option, about 15 sec... what in the world is going on?

Comment (jmythms) [reply src/EnergyPlus/ScheduleManager.cc:565]: Probably one is develop and the other one is release config. And probably you have a much better computer than I do 😄

**Review by mitchute — APPROVED** [review]
OK, well, this is definitely one we need to look into further. The current code isn't going to handle the errors, so we need to get this in now, but this should be on the short list of performance-related tasks that should be investigated.

File: src/EnergyPlus/ScheduleManager.cc (hunk 2 above)
Comment (jmythms) [reply src/EnergyPlus/ScheduleManager.cc:565]: I tested both in release configuration.

Comment (mitchute) [reply src/EnergyPlus/ScheduleManager.cc:565]: Nope, I double-checked, rebuilt, and ran multiple times. I'm in release mode on Mac. With ```UtilityRoutines::ProcessNumber``` this file is taking about 2:30-3:00 min; with ```std::stod```, it's about 15 sec.

Comment (jmythms) [reply src/EnergyPlus/ScheduleManager.cc:565]: That's very interesting... For me, Linux release build on CLion, `UtilityRoutines::ProcessNumber` takes 8 seconds vs `std::stod` takes 5ish.

**Comment by mitchute** [comment]
No diffs here, and I just ran unit tests locally to confirm that the error on the last run isn't real. Looks good. Merging. I'll open a new issue to investigate getting rid of ```ProcessNumber```.

**Comment by jmythms** [comment]
Thank you!

**Pull request merged**